spec/integration/client/fips_spec.rb

```diff
@@ -0,0 +1,8 @@
+describe "chef-client fips" do
+  def enable_fips_if_supported
+    OpenSSL.fips_mode = true if ENV["OMNIBUS_FIPS_MODE"]
```

Comment:
@jaymzh well, we exposed a bug in my environment checking logic (this env is set to _false_ in non-FIPS environments when building via omnibus)